modules/react/modal/lib/ModalBody.tsx

```diff
@@ -0,0 +1,12 @@
+import * as React from 'react';
+
+import {createSubcomponent, ExtractProps} from '@workday/canvas-kit-react/common';
+import {Card} from '@workday/canvas-kit-react/card';
+import {Popup, usePopupModel} from '@workday/canvas-kit-react/popup';
+
+export const ModalBody = createSubcomponent('div')({
+  displayName: 'Modal.Body',
+  modelHook: usePopupModel,
+})<ExtractProps<typeof Card.Body>>(elemProps => {
+  return <Popup.Body overflowY="auto" padding={'l'} paddingTop="zero" {...elemProps} />;
```

Comment:
I noticed the vertical spacing of a "default" Modal looks a bit off:

<img width="465" alt="default-modal" src="https://user-images.githubusercontent.com/146020/186286875-b4d17eb3-64d2-4770-b241-24430813a422.png">

Note the large vertical space between the Modal Body and the Modal Footer (`48px` vs `24px` in the spec). Even the vertical space between the Modal Heading and Modal Body is smaller than it should be (`16px` vs `24px` in the spec) -- this issue existed even before this PR.

I'm using the basic version of the Modal to render this example with almost no props applied to any of the subcomponents:

```
<Modal.Card>
  <Modal.CloseIcon aria-label="Close" />
  <Modal.Heading>Test</Modal.Heading>
  <Modal.Body>
    This is a test.
  </Modal.Body>
  <Modal.Footer>
    <Modal.CloseButton as={DeleteButton}>
      Delete
    </Modal.CloseButton>
    <Modal.CloseButton>Cancel</Modal.CloseButton>
  </Modal.Footer>
</Modal.Card>
```

What do you think about adjusting the default padding of the Modal subcomponents to match the `24px` of vertical spacing between the Header, Body, and Footer from the spec? I see there's a bottom padding on ModalBody Popup.Body, but that the top padding has been set to `zero`. Seems like we should add top padding to prevent content at the top of the Body from being cut off as well? We could add `paddingY="m"` to the ModalBody and remove all other vertical padding/margin from the subcomponents.

Reply:
That's perfect, thanks for that suggestion James!

Reply:
We're going with Alan's approach instead (see top-level PR comment).